src/util/Constants.js

```diff
@@ -485,8 +485,13 @@ exports.ChannelTypes = {
   CATEGORY: 4,
   NEWS: 5,
   STORE: 6,
+  STAGE: 13,
 };
 
+for (const [key, value] of Object.entries(exports.ChannelTypes)) {
+  exports.ChannelTypes[value] = key;
+}
```

Comment:
Could make use of the [`createEnum()`](https://github.com/amishshah/discord.js/blob/feat/stage-channels/src/util/Constants.js#L739-L747) function here. The lines [above](https://github.com/amishshah/discord.js/blob/feat/stage-channels/src/util/Constants.js#L480-L489) of the `exports.ChannelTypes` declaration would need to be deleted as well (GitHub wouldn't let me suggest changes that far up)

```suggestion
exports.ChannelTypes = createEnum([
  'TEXT',
  'DM',
  'VOICE',
  'GROUP',
  'CATEGORY',
  'NEWS',
  // 6
  'STORE',
  ...Array(6).fill(null),
  // 13
  'STAGE',
]);
```